core/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 _YYYY-MM-DD_
 
 - Dependency loop error now lists all test cases that formed a cycle
+- Dependencies can now be defined pattern-free with `sequentialTestGroup`
```

Comment:
Please describe in changelog, which entities were added and which has been changed.